src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
```

Comment:
Do not create Lottery 3 times
![image](https://user-images.githubusercontent.com/20326858/228826792-2a76f3f0-4bee-4c49-9203-81d45a9f9d0e.png)